jbrowse/src/client/JBrowse/Browser/plugins/VariantPlugin/VariantWidget/VariantWidget.js

```diff
@@ -0,0 +1,180 @@
+                    <Table className={classes.table}>
+                        <TableHead>
+                            <TableRow>
+                                {linesJSX}
+                            </TableRow>
+                        </TableHead>
+                    </Table>
+               </div>
+            </BaseCard>
+        )
+    }
+
+
+    function VariantTable(feat) {
+        const classes = useStyles()
+        if (!feat["INFO"]["CLNREVSTAT"]){
+          return null
+        }
+        var headers = []
+        var rows = []
+        for (let i = 0; i<feat["INFO"]["CLNREVSTAT"].length; i++){
+          headers.push(<TableCell>{i}</TableCell>)
+          rows.push(<TableCell>{feat["INFO"]["CLNREVSTAT"][i]}</TableCell>)
+        }
+
+        return (
+          <BaseCard title="CLNREVSTAT">
+            <div style={{ width: '100%', maxHeight: 600, overflow: 'auto' }}>
+                <Table className={classes.table}>
+                    <TableHead>
+                        <TableRow>
+                            {headers}
+                        </TableRow>
+                    </TableHead>
+                    <TableBody>
+                        {rows}
+                    </TableBody>
+                </Table>
+            </div>
+          </BaseCard>
+        )
+    }
+
+    function makeDisplays(feat, displays){
+        var propertyJSX = []
+        for(var display in displays){
+            var tempProp = []
+            for(var property in displays[display].properties){
+                if(feat["INFO"][displays[display].properties[property]]){
+                    tempProp.push(
+                        <TableRow>
+                            {displays[display].properties[property]}: {feat["INFO"][displays[display].properties[property]]}
+                        </TableRow>
+                    )
+                }
+            }
+            propertyJSX.push(tempProp)
+        }
+
+        var displayJSX = []
+        for(var i = 0; i < displays.length; i++){
+            displayJSX.push(
+                <BaseCard title={displays[i].name}>
+                   <div style={{ padding: '7px', width: '100%', maxHeight: 600, overflow: 'auto' }}>
+                        <Table className={displays[i].name}>
+                            <TableHead>
+                                {propertyJSX[i]}
+                            </TableHead>
+                        </Table>
+                   </div>
+                </BaseCard>
+            )
+        }
+        return displayJSX
+    }
+
+    function NewTable(props) {
+        const classes = useStyles()
+        const { feature } = props
+        const { model } = props
+        const feat = JSON.parse(JSON.stringify(model.featureData))
+        var displays;
+
+
+        var MCJSX = MC(feat)
+        if(MCJSX){
+            feat["INFO"]["MC"] = null
+        }
+
+        var CLNREVSTAT = VariantTable(feat)
```

Comment:
Is there a reason we need to special-case specific INFO field properties by name here? The hope is that the caller can provide a config object, and all code within the plugin is completely unaware of specific property names.

Reply:
No, that functionality is all covered by the current implementation. This is a holdover from prior iterations, will remove.

Reply:
updated